templates/web/fixmystreet.com/about/press

```diff
@@ -0,0 +1,46 @@
+[% INCLUDE 'header.html',
+    title = loc('Press'),
+    bodyclass = 'twothirdswidthpage' %]
+
+[% INCLUDE 'about/_sidebar.html' %]
+
+<h1>Press resources</h1>
+
+<h2>About FixMyStreet</h2>
+
+<p>
+    FixMyStreet is an independent web service run by the charity <a href="https://www.mysociety.org/">mySociety</a> to make it easier for anyone to report a problem in their local area without needing to have any knowledge of council boundaries or responsibilities.
+</p>
+
+<p>
+You can find the answers to many of our most frequently asked questions on <a href="https://www.fixmystreet.com/faq">our FAQ page</a>.
+</p>
+
+<h2>How to contact us</h2>
+
+<p>
+    Whether you’re looking for a quote, an interview, some images, or something else entirely, the mySociety press team is happy to help you with your story.
+</p>
+
+<p>
+If you're looking for statistics, we welcome enquiries about and support the use of FixMyStreet data for constructive research purposes. 
+</p>
+
+<p>
+<b>Please note</b>, mySociety does not endorse the simplistic use of FixMyStreet data to <b>compare, denounce or rank</b> areas. This is because the data from FixMyStreet alone cannot be used to make fair comparisons at the national level. Please read our statement on this. 
+</p>
+
+<p>
+We are a small, fully remote organisation, so the best way to reach us is to email <b>press@mysociety.org</b> and one of the team will respond to you as quickly as possible. 
```

Comment:
We generally, for anti-spam reasons that don't really work any more, but hey, put `&#64;` instead of `@`, which is the same.